test/python/compiler/test_assembler.py

```diff
@@ -707,6 +696,15 @@ def test_assemble_parametric_unsupported(self):
         qobj_insts = qobj.experiments[0].instructions
         self.assertFalse(hasattr(qobj_insts[0], 'pulse_shape'))
 
+    def test_init_qubits(self):
+        """Check that the ``init_qubits`` assemble option is passed on to the qobj."""
+        qobj = assemble(self.schedule, self.backend)
+        self.assertRaises(AttributeError, getattr, qobj.config, 'init_qubits')
+        qobj = assemble(self.schedule, self.backend, init_qubits=True)
+        self.assertEqual(qobj.config.init_qubits, True)
+        qobj = assemble(self.schedule, self.backend, init_qubits=False)
+        self.assertEqual(qobj.config.init_qubits, False)
```

Comment:
I would break this tests. Failing some of the subtests and not all might give information about what went wrong.
```suggestion
    def test_init_qubits_default(self):
        """Check that the init_qubits=None assemble option is passed on to the qobj."""
        qobj = assemble(self.schedule, self.backend)
        self.assertRaises(AttributeError, getattr, qobj.config, 'init_qubits')

    def test_init_qubits_true(self):
        """Check that the init_qubits=True assemble option is passed on to the qobj."""
        qobj = assemble(self.schedule, self.backend, init_qubits=True)
        self.assertEqual(qobj.config.init_qubits, True)
  
    def test_init_qubits_false(self):
        """Check that the init_qubits=False assemble option is passed on to the qobj."""
        qobj = assemble(self.schedule, self.backend, init_qubits=False)
        self.assertEqual(qobj.config.init_qubits, False)
```

Reply:
Very, very good point. Thank you for creating a diff for me!! :D